CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG for Sulu
 ==================
 
+* dev-master
+    * HOTFIX      #1386 [ContentBundle]  Fixed hard-coded values in search metadata
```

Comment:
Should probably be something like: Correct search indexing of content blocks on pages.